app/views/campaigns/_form.html.erb

```diff
@@ -162,14 +152,20 @@
       <% end %>
     <% end %>
     <%= card_body do %>
-      <div class="row" data-controller="campaignForm">
-        <div data-target="campaign-form.creativesWrapper">
-          <%= render partial: "/creative_options/options",
-              locals: {
-                selected_creative_ids: campaign.creative_ids,
-                creatives: campaign.user.creatives.active.standard.order(name: :asc)
-              } %>
-        </div>
+      <div class="d-flex flex-column">
+        <% Current.organization.creatives.active.each do |creative| %>
+          <div class="form-check">
+            <%= tag.input type: "checkbox",
+                          class: "form-check-input",
+                          value: creative.id,
+                          name: "campaign[creative_ids][]",
+                          id: "campaign_creative_id_#{creative.id}",
+                          checked: campaign.creatives.include?(creative) %>
+            <label class="form-check-label" for="campaign_creative_id_<%= creative.id %>">
+              <%= creative.name %> - <strong><%= creative.headline %></strong> <%= creative.body %>
+            </label>
+          </div>
+        <% end %>
```

Comment:
What are the semantics around *active creative*? I thought it meant, creative that is currently participating in an active campaign.